question/question.css

```diff
@@ -38,10 +42,13 @@ main>* {
 }
 
 li.wrong{
-    background-color: var(--invalid-red);
+    background-color: var(--error-color);
 }
 
 li.right{
-    background-color: var(--valid-green);
-    color: var(--black);
-}
\ No newline at end of file
+    background-color: #4FBA52;
+}
+
```

Comment:
suggestion:  you can define this color into variable

Reply:
yeah, forgot to do it, thanks!